html/elements/menulist.json

```diff
@@ -0,0 +1,476 @@
+{
+  "html": {
+    "elements": {
+      "menulist": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/menulist",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "8",
+              "notes": [
+                "Only works for <code>&lt;menuitem&gt;</code> elements defined within a <code>&lt;menu&gt;</code> element assigned to an element via the <code>contextmenu</code> attribute.",
+                "The &lt;menuitem&gt; element requires a closing tag."
```

Comment:
`The <code>&lt;menuitem&gt;</code>` (here and throughout the file)

Reply:
Fixed.